app/plugins/router.js

```diff
@@ -29,7 +29,8 @@ const routes = [].concat(
   require('../routes/vet/practice'),
   require('../routes/vet/rcvs'),
   require('../routes/vet/reference'),
-  require('../routes/vet/visit-date')
+  require('../routes/vet/visit-date'),
+  require('../routes/vet/beef-eligibility')
```

Comment:
Can this be added in alphabetical order please? It helps when looking at the routing to see what is there.